idseq_pipeline/commands/common.py

```diff
@@ -211,6 +211,43 @@ def __exit__(self, *args):
             self.tail_subproc.kill()
 
 
+class AsyncHandler:
+    def __init__(self):
+        self.threads = []
+        pass
+
+    def launch(self, target, args):
+        if type(args) == str:
+            args = (args, )
```

Comment:
Let's keep this to straight args, matching the threading and multiprocessing interfaces, and more streamlined.